src/services/EventHandler.ts

```diff
@@ -15,6 +15,8 @@ export class EventHandler {
 
   public async runHandler() {
     const result = await this.splitwiseClient.getExpenses();
+    logger.info(`Found ${result.expenses.length} expenses!`);
```

Comment:
Don't really see the point on this logging line. As we are getting all user expenses on Splitwise, this includes payments, deleted, settled since the beggining of time for the user :) this means that I get 100 transaction out of which none if going to be processed